docs/gis-usage.md

```diff
@@ -154,20 +154,41 @@ GIBS imagery layers can be imported into [ESRI's ArcGIS Online](http://www.arcgi
 
 ### Adding GIBS layers via WMS
 
-* Click the "Modify Map" button in the upper right corner, then "Add | Add Layer from Web".
+* Click on "Map" in the top menu navigation
+
+![ArcGIS Online, Access Map](img/AGOL-AccessMap.png)
+
+* Click on the down arrow next to the "Add" icon in the left side Layers section, then "Add Layer from URL"
 
 ![ArcGIS Online, Add Layer](img/AGOL-AddLayer.png)
 
-* Change the data type to a WMS OGC Web Service
-* Paste one of the [GIBS WMS endpoints](../access-basics/#ogc-web-map-service-wms) (e.g., [WGS84 / Geographic](https://gibs.earthdata.nasa.gov/wms/epsg4326/best/wms.cgi) (e.g., [WGS84 / Geographic](https://gibs.earthdata.nasa.gov/wms/epsg4326/best/wms.cgi) into the "URL" field and press the "Get Layers" button
-* Deselect all of the layers (upper left "square") and select a layer you'd like to add
+* Or click on the plus button in the far left menu navigation bar, then "Add Layer from URL"
+
+![ArcGIS Online, Add Layer from URL](img/AGOL-AddLayer2.png)
+
+* Paste one of the [GIBS WMS endpoints](../access-basics/#ogc-web-map-service-wms) into the "URL" field; note that the [GIBS EPSG:3857 WMS endpoint](https://gibs.earthdata.nasa.gov/wms/epsg3857/best/wms.cgi) works best since this is the native projection of the ArcGIS Online map.
```

Comment:
This linked GIBS URL loads a page that says "No query information to decode. QUERY_STRING is set, but empty". I realize this is because there are no appended parameters but it may be misleading to new users.

Reply:
I removed the hyperlinks here and elsewhere on the page.